.cci.jenkinsfile

```diff
@@ -0,0 +1,17 @@
+// Documentation: https://github.com/coreos/coreos-ci/blob/master/README-upstream-ci.md
+
+cosaPod(buildroot: true) {
+    checkout scm
+
+    fcosBuild(make: true)
+
+    stage("Build metal") {
+        shwrap("cosa build metal")
```

Comment:
Oh right, so one thing here is that the FCOS build actually happens in `/srv/fcos` which is on an `emptyDir`. Easiest is to just wrap all the additional stages in a `dir("/srv/fcos") { ... }`.